.gitignore

```diff
@@ -17,3 +17,10 @@ project.lock.json
 /.dotnet/
 tools/**/obj/
 tools/**/build/
+
+# Docs artifacts
+docs/obj/
+docs/bin/
+
+# Xml doc/test files
+*.xml
```

Comment:
I don't think ignoring all XML files everywhere is a good idea, this isn't needed anymore anyway.

Reply:
Mind you, this is only the root folder. It also prevents `DesktopClrTestResults.xml` and `NetCoreClrTestResults.xml` from being unstaged and accidentally committed.